src/containerapp/azext_containerapp/tests/latest/test_containerapp_dapr_resiliency.py

```diff
@@ -30,7 +30,7 @@ def test_containerapp_resiliency(self, resource_group):
         bad_capp = "bad-capp"
         resil_policy_count = 1
         
-        create_containerapp_env(self, env_name, resource_group)
+        create_containerapp_env(self, env_name, resource_group, location=TEST_LOCATION)
```

Comment:
No need to input `location=TEST_LOCATION` here, because TEST_LOCATION is the default location with following configure command:
`self.cmd('configure --defaults location={}'.format(TEST_LOCATION))`

Reply:
That was there because to run in Dogfood region against my own RP I needed to provide a specific region - this is why I actually had changed the other utility function also.

I can remove it.

I think we do need some changes in the future so that it is possible to run against PPE / Dogfood regions more easily.